semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/messaging/MessagingSemanticConventionUtils.java

```diff
@@ -37,4 +46,43 @@ public static boolean isRabbitMqBackend(Event event) {
     return SpanAttributeUtils.containsAttributeKey(event, RawSpanConstants.getValue(RabbitMq.RABBIT_MQ_ROUTING_KEY))
         || SpanAttributeUtils.containsAttributeKey(event, OtelMessagingSemanticConventions.RABBITMQ_ROUTING_KEY.getValue());
   }
+
+  public static Optional<String> getKafkaBackendURI(Event event) {
+    if (!isKafkaBackend(event)) {
+      return Optional.empty();
+    }
+
+    if(SpanAttributeUtils.containsAttributeKey(event, MESSAGING_URL)) {
+      return Optional.of(SpanAttributeUtils.getStringAttribute(event, MESSAGING_URL));
+    }
+    return DbSemanticConventionUtils.getBackendURIForOtelFormat(event);
+  }
+
+  public static boolean isKafkaBackend(Event event) {
+    if(SpanAttributeUtils.containsAttributeKey(event, MESSAGING_SYSTEM)) {
+      return KAFKA_SYSTEM_VALUE.equals(SpanAttributeUtils.getStringAttributeWithDefault(
```

Comment:
It seems that `peer.service` is used with open-tracing spans tags. So, we should also add that along with open-telemetry tags - https://github.com/open-telemetry/opentelemetry-specification/blob/1afab39e5658f807315abf2f3256809293bfd421/specification/trace/semantic_conventions/messaging.md#apache-kafka-example

https://objectpartners.com/2019/04/25/distributed-tracing-with-apache-kafka-and-jaeger/

@rish691 @Aka-shi @jcchavezs Let me know what do you guys think?

Reply:
`peer.service` is the opentracing way of say "remoteEndpoint" (see https://github.com/openzipkin-contrib/zipkin-go-opentracing/blob/master/tracer.go#L120) so yeah we probably want to support it.

Reply:
Added basic support for Open tracing semantics. Didn't extend this for other backends, will do that in a separate PR.